src/main/java/Duke.java

```diff
@@ -1,10 +1,105 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        introMessage();
+
+        String reply;
+        Scanner scan = new Scanner(System.in);
+
+        // Array of the Task class objects
+        Task[] list = new Task[100];
+
+        int listCounter = 0;
+
+        String taskName;
+        String dueDate;
+
+        int TODO_REMOVAL = 5;
+        int DEADLINE_REMOVAL = 9;
+        int EVENT_REMOVAL = 6;
+
+        while (true) {
+            reply = scan.nextLine();
+
+            if (reply.equals("bye")) {
+                outroMessage();
+                break;
+            } else if (reply.equals("list")){
+                printList(list, listCounter);
+            } else if (reply.contains("done")){
+                for (int j = 1; j <= listCounter ; j++){
+                    if (reply.contains(Integer.toString(j))){ //finding out the task that is chosen
+                        list[j-1].markAsDone(); // minus 1 coz the array of Task is 1 digit ahead
+                    }
+                }
+            } else if (reply.contains("todo")){
+                taskName = reply.substring(TODO_REMOVAL);
+                Todo newTask = new Todo(taskName);
+                listCounter = addedList(list, listCounter, newTask);
+            } else if (reply.contains("deadline")){
+                taskName = reply.substring(DEADLINE_REMOVAL,reply.indexOf("/") - 1);
+                dueDate = reply.substring(reply.indexOf("/by") + 4);
+                Deadline newTask = new Deadline(taskName,dueDate);
+                listCounter = addedList(list, listCounter, newTask);
+            } else if (reply.contains("event")){
+                taskName = reply.substring(EVENT_REMOVAL,reply.indexOf("/") - 1);
+                dueDate = reply.substring(reply.indexOf("/at") + 4);
+                Event newTask = new Event(taskName,dueDate);
+                listCounter = addedList(list, listCounter, newTask);
+            } else {
```

Comment:
Can consider changing this into a switch statement. It's clearer when you have more commands.

Reply:
Thanks for the advice, will consider changing this